### PR TITLE
chore: Add background and border for resource card (#2463)

### LINKED ISSUE #2463: [OTHER] Add background color in the resource card in light theme
### What would you like to share or ask?

The resource card have no background color on light theme.

image

Add `light-gray` background color and a slight `border` same as the category page.

image

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [ ] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by AlexMutig**
## Changes proposed

Add background and border for resource card

## Screenshots

Screenshot 2024-08-09 182847

## Note to reviewers

I hope i did everything right. If not, I am eager to fix it.

### CODE REVIEW

File: components/Cards/Card.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const Card: FC<CardProps> = ({ data, onClick }) => {
 
   return (
     <article className="z-10 h-full w-full rounded-3xl dark:bg-slate-800 dark:text-text-primary dark:shadow-sm bg-light-white">
-      <div className="card-body">
+      <div className="card-body border border-theme-secondary/25 rounded-xl bg-theme-primary-light dark:border dark:border-theme-primary dark:border-opacity-8 dark:bg-theme-tertiary">
```
Comment (Jay-Karia) [inline components/Cards/Card.tsx:32]: IMO, class `rounded-xl` is not required

Comment (AlexMutig) [reply components/Cards/Card.tsx:32]: Alright, I will remove it. Shall i create another pull request after this?

Comment (AlexMutig) [reply components/Cards/Card.tsx:32]: Actually i just double checked and if i remove the rounded-xl class the cards aren't rounded anymore.
Screenshot 2024-08-10 150843

Comment (Jay-Karia) [reply components/Cards/Card.tsx:32]: No, just push commit removing the class in your branch.
The pull will be updated.

Comment (AlexMutig) [reply components/Cards/Card.tsx:32]: I made the cards look like the ones in the category page as requested. If a maintainer asks me to remove it i will. I will keep it for now.

Comment (Jay-Karia) [reply components/Cards/Card.tsx:32]: There's another class `rounded-3xl` in `<article>` which is responsible for border-radius

Comment (AlexMutig) [reply components/Cards/Card.tsx:32]: Alright i will look at it soon. Thank you!

Comment (AlexMutig) [reply components/Cards/Card.tsx:32]: Ok i looked at it and i need to write the border classes into the  `<article>`  and change the bg-color for it to work like i want it to. Is this right?.

Comment (Jay-Karia) [reply components/Cards/Card.tsx:32]: Yes

Comment (AlexMutig) [reply components/Cards/Card.tsx]: Ok i updated it. This should hopefully be ok now.

**Review by Jay-Karia — APPROVED** [review]

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**